installation/citus_cloud.rst

```diff
@@ -1,23 +1,18 @@
 .. _multi_node_cloud:
 
-Managed Deployment
+Managed Service
 ==================
 
 One of the easiest ways to get started with Citus is to spin up a cluster in
-the cloud on Azure. Citus is available as a built-in deployment option in the
-`Azure Database for PostgreSQL - Hyperscale (Citus)
-<https://docs.microsoft.com/azure/postgresql/hyperscale/>`_ managed
-service.
+the cloud on Azure. The Citus managed service is available as
+`Azure Cosmos DB for PostgreSQL
+<https://learn.microsoft.com/azure/cosmos-db/postgresql/introduction/>`_, formerly known as Hyperscale (Citus) in Azure Database for PostgreSQL.
 
-To get started, you can take advantage of our Hyperscale (Citus) `quickstart
+To get started, you can take advantage of our Citus on Azure `quickstart
```

Comment:
Might be best to call it the "Azure Cosmos DB for PostgreSQL quickstart" rather than "Citus on Azure quickstart." The naming has been confusing and variable as it is. 😉

Reply:
will do, although I propose we keep the connection to "Citus on Azure" in the sentence while at the same time referring to the quickstart as you suggest. I believe the Citus on Azure connection is important because the new name doesn't make it intuitive that "Azure Cosmos DB for PostgreSQL" gets you to Citus. And the user is reading the Citus open source docs at this point.